semgrep-core/tests/OTHER/rules/inception2.yaml

```diff
@@ -0,0 +1,24 @@
+rules:
+#ERROR: double-id!
+- id: double-id
+  message: |
+    you use multiple times the same id
+  severity: ERROR
+  languages: [json, yaml]
+  patterns:
+    - pattern-inside: |
+       [{id: $X, ...}, ..., {id: $X, ...}, ...]
```

Comment:
I'm able to do it by using JSON to specify the pattern, but the yaml pattern should parse the same.